hapi/hedera-protobufs/services/consensus_create_topic.proto

```diff
@@ -70,4 +71,55 @@ message ConsensusCreateTopicTransactionBody {
      * If specified, there must be an adminKey and the autoRenewAccount must sign this transaction.
      */
     AccountID autoRenewAccount = 7;
+
+    /**
+     * Access control for update or delete of custom fees.
+      * <br/>
+     * If set, subsequent consensus_update_topic transactions signed with this
+     *  key MAY update or delete the custom fees for this topic. If not set,
+     *  the custom fees for this topic are immutable.
+     * <p>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * <br/>
+     * If set when the topic is created, this field MAY be changed via update.
+     */
+    Key fee_schedule_key = 8;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <br/>
+     * If a submit transaction is signed by _any_ key included in this set,
+     *  custom fees SHALL NOT be charged for that transaction.
+     * <p>
+     * This field SHALL NOT contain more than
+     *  `MAX_ENTRIES_FOR_FEE_EXEMPT_KEY_LIST` keys.
+     * <br/>
+     * fee_exempt_key_list SHALL NOT contain any duplicate keys.
+     * <br/>
+     * fee_exempt_key_list MAY contain keys for accounts that are inactive,
+     *  deleted, or non-existent.
```

Comment:
A line break tag is needed at the end of this line.

Reply:
Fixed